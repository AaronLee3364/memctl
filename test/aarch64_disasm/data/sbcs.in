fffffff006366a44 fa110210
