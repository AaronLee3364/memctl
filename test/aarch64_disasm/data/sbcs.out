fffffff006366a44  SBCS    X16, X16, X17
